src/einsteinpy/utils/kerr_utils.py

```diff
@@ -331,3 +334,87 @@ def nonzero_christoffels():
         if chl[i][k][l]:
             vcl.append((i, k, l))
     return vcl
+
+
+def spin_factor(J, M, c):
+    """
+    Calculate spin factor(a) of kerr body
+
+    Parameters
+    ----------
+    J : float
+        Angular momentum in SI units(kg m2 s-2)
+    M : float
+        Mass of body in SI units(kg)
+    c : float
+        Speed of light
+
+    Returns
+    -------
+    float
+        Spin factor (J/(Mc))
+    
+    """
+    return J / (M * c)
+
+
+def event_horizon(Rs, a, theta=constant.pi_by_2, coord="BL"):
+    """
+    Calculate the radius of event horizon of Kerr black hole
+
+    Parameters
+    ----------
+    Rs : float
+        Schwarzschild Radius
+    a : float
+        Black hole spin factor
+    theta : float
+        Angle from z-axis in Boyer-Lindquist coordinates in radians. Mandatory for coord=='Spherical'.
+    coord : str
+        Output coordinate system. 'BL' for Boyer-Lindquist & 'Spherical' for spherical. Defaults to 'BL'.
+
+    Returns
+    -------
+    ~numpy.array
+        [Radius of event horizon(R), angle from z axis(theta)] in BL/Spherical coordinates
+    
+    """
+    Rh = 0.5 * (Rs + np.sqrt((Rs ** 2) - 4 * (a ** 2)))
+    if coord == "BL":
+        ans = np.array([Rh, theta], dtype=float)
+    else:
+        ans = utils.CartesianToSpherical_pos(
+            utils.BLToCartesian_pos(np.array([Rh, theta, 0.0]), a)
+        )[:2]
+    return ans
+
+
+def ergosphere(Rs, a, theta=constant.pi_by_2, coord="BL"):
```

Comment:
Rename it to radius_ergosphere for more clarity!

Reply:
fair enough!